jdi-light-vuetify-tests/src/main/java/io/github/com/pages/BadgesPage.java

```diff
@@ -13,14 +13,14 @@ public class BadgesPage extends VuetifyPage {
     @UI("#simpleBadges .v-badge__badge")
     public static List<Badge> simpleBadges;
 
-    @UI("//span[contains(text(), 'Unlock Account')]")
-    public static Button unlockAccountButton;
+    @UI("//span[contains(text(), 'Unlock Account') or contains(text(), 'Lock account')]")
+    public static Button lockUnlockAccountButton;
 
     @UI("//span[contains(text(), 'Lock account')]")
```

Comment:
In case of previous change this button is not needed

Reply:
Done by removing.